### PR TITLE
fix linux string utils

### PR DESCRIPTION
**Pull request by anporumb**

### CODE REVIEW

File: linux/inc/c_pal/string_utils.h [hunk 1]
```diff
@@ -1,62 +1,64 @@
...
+#include "macro_utils/macro_utils.h"
+
+#include "umock_c/umock_c_prod.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*produces a string as if printed by printf*/
+extern char* sprintf_char_function(const char* format, ...);
```
Comment (dcristoloveanu) [inline linux/inc/c_pal/string_utils.h:31]: >extern [](http://example.com/codeflow?start=0&length=6)

Likely not needed

Comment (anporumb) [reply linux/inc/c_pal/string_utils.h:31]: gzzzzzzzz

**Review by dcristoloveanu — APPROVED** [review]
:shipit:

File: linux/inc/c_pal/string_utils.h (hunk 1 above)
Comment (anporumb) [reply linux/inc/c_pal/string_utils.h]: removed from here and everywhere

**Pull request merged**